src/actions/guides/getting-started/installing.cr

```diff
@@ -1,5 +1,6 @@
 class Guides::GettingStarted::Installing < GuideAction
   ANCHOR_INSTALL_REQUIRED_DEPENDENCIES = "perma-install-required-dependencies"
+  ANCHOR_POSTGRESQL                    = "perma-install-postgres"
```

Comment:
For this to work you'll also need to use `#{permalink(ANCHOR_POSTGRESQL)` above the heading on 211. See example on line 11